R/ping.R

```diff
@@ -12,6 +12,8 @@
 #' }
 ping <- function(url = get_ckanr_url(), as="logical", ...)
 {
-  res <- ckan_POST(url, 'site_read', ...)
-  switch(as, json = res, logical = jsd(res))
+  retval <- tryCatch({
```

Comment:
@wush978 nothing is returned from this function though, you assign to `retval` , so the value isn't returned